packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { AMPJSON } from './AMPJSON';
+import { css, cx } from 'emotion';
+import Tick from '@guardian/pasteup/icons/tick.svg';
+
+const fontFamily =
+    'GuardianTextSans,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif';
+
+const consentUIStyle = css`
+    font-family: ${fontFamily};
+    font-size: 1rem;
+    line-height: 1.25rem;
+    color: #f6f6f6;
+    background-color: rgb(51, 51, 51);
+
+    max-width: 37.5rem;
```

Comment:
Would it be possible to use `px` instead of `rem`, as they are easier for devs to work with?

Reply:
I copied the style from the body. Styling is a pain point for me in .component :/

Reply:
Okay. Let's sort that out!